cove_iati/lib/iati.py

```diff
@@ -421,3 +426,21 @@ def get_file_type(file):
         return 'csv'
     else:
         raise UnrecognisedFileTypeXML
+
+
+def iati_identifier_count(data_file):
+    tree = get_tree(data_file)
+    root = tree.getroot()
+    identifiers = root.xpath('/iati-activities/iati-activity/iati-identifier/text()')
+    unique_identifiers = list(set(identifiers))
+
+    return len(unique_identifiers)
+
+
+def organisation_identifier_count(data_file):
+    tree = get_tree(data_file)
```

Comment:
I think it would be better to avoid opening the file multiple times, by passing around the tree object.

Reply:
Yes, you are right. I have done an amendment to that.